### PR TITLE
revert overlays to use charmstore charms until a 1.24 release 

### PR DESCRIPTION
**Pull request by addyess**
at date, charmhub CDK charms haven't reached a stable release on charmhub and therefore shouldn't be included

Evidence that the following overlays can be applied to the current latest/stabe `charmed-kubernetes` bundle 
https://paste.ubuntu.com/p/yysQbGZVRY/

### CODE REVIEW

File: overlays/ceph-fs.yaml [hunk 1]
```diff
@@ -10,12 +10,12 @@ applications:
       osd-devices: '32G,2'
       osd-journals: '8G,1'
   ceph-mon:
-    charm: cs:ceph-mon
+    charm: ceph-mon
```
Comment (addyess) [inline overlays/ceph-fs.yaml:13]: a `latest/stable` of this charm exists in charmhub

File: overlays/equinix-overlay.yaml [hunk 2]
```diff
@@ -36,7 +36,7 @@ applications:
     - lxd:1
     - lxd:2
   ceph-fs:
-    charm: cs:ceph-fs
+    charm: ceph-fs
```
Comment (addyess) [inline overlays/equinix-overlay.yaml:39]: a `latest/stable` of this charm exists in charmhub

File: overlays/equinix-overlay.yaml [hunk 3]
```diff
@@ -60,7 +60,7 @@ applications:
      - lxd:1
      - lxd:2
   ceph-osd:
-    charm: cs:ceph-osd
+    charm: ceph-osd
```
Comment (addyess) [inline overlays/equinix-overlay.yaml:63]: a `latest/stable` of this charm exists in charmhub

File: overlays/logging-egf-overlay.yaml [hunk 4]
```diff
@@ -1,36 +1,36 @@
...
   apache2:
-    charm: cs:apache2
+    charm: apache2
     num_units: 1
     expose: true
     options:
       enable_modules: "headers proxy_html proxy_http"
   elasticsearch:
-    charm: cs:elasticsearch
+    charm: elasticsearch
```
Comment (addyess) [inline overlays/logging-egf-overlay.yaml:9]: a `latest/stable` of this charm exists in charmhub

File: overlays/logging-egf-overlay.yaml [hunk 5]
```diff
@@ -1,36 +1,36 @@
...
     constraints: mem=7G root-disk=16G
     num_units: 1
     options:
       apt-repository: "deb https://artifacts.elastic.co/packages/6.x/apt stable main"
   filebeat:
-    charm: cs:filebeat
+    charm: filebeat
```
Comment (addyess) [inline overlays/logging-egf-overlay.yaml:15]: a `latest/stable` of this charm exists in charmhub

File: overlays/monitoring-pgt-overlay.yaml [hunk 6]
```diff
@@ -1,18 +1,18 @@
 applications:
   prometheus:
-    charm: cs:prometheus2
+    charm: prometheus2
```
Comment (addyess) [inline overlays/monitoring-pgt-overlay.yaml:3]: a `latest/stable` of this charm exists in charmhub

File: overlays/monitoring-pgt-overlay.yaml [hunk 7]
```diff
@@ -1,18 +1,18 @@
...
     constraints: "mem=4G root-disk=16G"
     num_units: 1
   grafana:
-    charm: cs:grafana
+    charm: grafana
```
Comment (addyess) [inline overlays/monitoring-pgt-overlay.yaml:7]: a `latest/stable` of this charm exists in charmhub

File: overlays/vault-pki-ha-overlay.yaml [hunk 8]
```diff
@@ -1,15 +1,15 @@
...
   easyrsa: null
   vault:
-    charm: cs:vault
+    charm: vault
     num_units: 2
     options:
       auto-generate-root-ca-cert: true
   percona-cluster:
-    charm: cs:percona-cluster
+    charm: percona-cluster
```
Comment (addyess) [inline overlays/vault-pki-ha-overlay.yaml:9]: a `latest/stable` of this charm exists in charmhub

File: overlays/vault-pki-overlay.yaml [hunk 9]
```diff
@@ -1,15 +1,15 @@
 applications:
   easyrsa: null
   vault:
-    charm: cs:vault
+    charm: vault
```
Comment (addyess) [inline overlays/vault-pki-overlay.yaml:4]: a `latest/stable` of this charm exists in charmhub

File: overlays/vault-pki-overlay.yaml [hunk 10]
```diff
@@ -1,15 +1,15 @@
...
     num_units: 1
     options:
       auto-generate-root-ca-cert: true
   percona-cluster:
-    charm: cs:percona-cluster
+    charm: percona-cluster
```
Comment (addyess) [inline overlays/vault-pki-overlay.yaml:9]: a `latest/stable` of this charm exists in charmhub

File: overlays/equinix-overlay.yaml [hunk 11]
```diff
@@ -7,7 +7,7 @@ machines:
     constraints: mem=32G
 applications:
   calico:
-    charm: cs:~containers/calico-826
+    charm: cs:~containers/calico
```
[the remark this answers is not in the document]
Comment (evilnick) [reply overlays/equinix-overlay.yaml:10]: There was at the time, I don't believe it is required though.

**Review by evilnick — APPROVED** [review]
:+1:

**Review by stonepreston — APPROVED** [review]
Everything looks good.  I double checked that all charms that do not have a cs prefix do in fact have stables available on charmhub, and also that no control plane relations are listed. Everything is master. I cant speak to the specific revision of calico in the equinix overlay however

File: overlays/equinix-overlay.yaml (hunk 11 above)
[the remark this answers is not in the document]
Comment (addyess) [reply overlays/equinix-overlay.yaml:10]: Thanks @evilnick , so this is ok with you or should it stay pinned to 826?

**Pull request merged**